debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -62,6 +62,10 @@ public class PostgresReplicationConnection extends JdbcConnection implements Rep
 
     private static final String SQL_STATE_INSUFFICIENT_PRIVILEGE = "42501";
 
+    private static final String SQL_LOCK_NOT_AVAILABLE = "55P03";
+
+    private static final String PUBLICATION_QUERY_FAILURE_MESSAGE = "Creation of publication slot failed: query to create/update publication timed out, please make sure that there are no maintenance activities going on the database end.";
```

Comment:
@Kalla-Shubham Let's update the message here:

> Creation of publication slot failed;
Creation/updation of publication failed: